crates/shielded_token/src/storage_key.rs

```diff
@@ -121,6 +121,15 @@ pub fn is_masp_commitment_anchor_key(key: &storage::Key) -> bool {
         ] if *addr == address::MASP && prefix == MASP_NOTE_COMMITMENT_ANCHOR_PREFIX)
 }
 
+/// Check if the given storage key is a masp token map key
+pub fn is_masp_token_map_key(key: &storage::Key) -> bool {
+    matches!(&key.segments[..],
+    [DbKeySeg::AddressSeg(addr),
+             DbKeySeg::StringSeg(prefix),
+             ..
```

Comment:
I think we can avoid this line since the token map key only contains the first two segments

Reply:
Thanks. Rectified.